tfhe/src/integer/server_key/radix_parallel/vector_find.rs

```diff
@@ -17,6 +17,7 @@ use std::usize;
 /// Outputs are not required to be unique
 /// Input values are not required to span all possible values that
 /// ` ct` could hold.
+#[derive(Debug)]
```

Comment:
Looks unrelated

Reply:
In the first commit of this PR, there is a fix, to find the data that triggered the error I added prints of the luts used in the assert in case the thing fails, and kept these in case it fails again later